target/linux/ath79/image/generic.mk

```diff
@@ -75,6 +75,17 @@ define Device/seama
   SEAMA_SIGNATURE :=
 endef
 
+define Device/airtight_c-65
+  ATH_SOC := qca9558
+  DEVICE_VENDOR := Airtight
+  DEVICE_MODEL := C-65
+  DEVICE_PACKAGES := kmod-ath10k ath10k-firmware-qca988x
+  IMAGE/default := append-kernel | pad-to $$$$(BLOCKSIZE) | append-rootfs | pad-rootfs | check-size $$$$(IMAGE_SIZE)
+  IMAGE_SIZE := 15936k
+  IMAGE/sysupgrade.bin := append-kernel | append-rootfs | pad-rootfs | append-metadata | check-size $$$$(IMAGE_SIZE)
```

Comment:
This is redefinition of image recipe in https://github.com/openwrt/openwrt/blob/d616b2c906690d2e471144ca12b0a9ed28de21c2/target/linux/ath79/image/Makefile#L65-L66 which differs only in padding kernel to block size (which generally should be done). Is there a reason for this?

Reply:
Removed